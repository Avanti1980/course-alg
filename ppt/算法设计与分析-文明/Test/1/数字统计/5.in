2
9999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999
10000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000