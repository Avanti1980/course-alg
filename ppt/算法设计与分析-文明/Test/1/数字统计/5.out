15337
15338